static/erdblick/debugapi.js

```diff
@@ -41,15 +43,24 @@ export class ErdblickDebugApi {
      */
     getCamera() {
         const position = [
-            this.mapView.viewer.camera.position.x,
-            this.mapView.viewer.camera.position.y,
-            this.mapView.viewer.camera.position.z
+            this.view.viewer.camera.position.x,
+            this.view.viewer.camera.position.y,
+            this.view.viewer.camera.position.z
         ];
         const orientation = {
-            heading: this.mapView.viewer.camera.heading,
-            pitch: this.mapView.viewer.camera.pitch,
-            roll: this.mapView.viewer.camera.roll
+            heading: this.view.viewer.camera.heading,
+            pitch: this.view.viewer.camera.pitch,
+            roll: this.view.viewer.camera.roll
         };
         return JSON.stringify({ position, orientation });
     }
+
+    /**
+     * Generate a test TileFeatureLayer, and show it.
+     */
+    showTestTile() {
```

Comment:
nice, being able to check basic visualization functionality with integrated test data is both good for initial testing as well as for regression testing :)